Sources/SourceKit/sourcekitd/SwiftLanguageServer.swift

```diff
@@ -79,6 +79,20 @@ public final class SwiftLanguageServer: ToolchainLanguageServer {
     response: SKResponseDictionary,
     for snapshot: DocumentSnapshot)
   {
+    // Disable diagnostics for virtual files. Some editors (like Visual Studio Code) use virtual
```

Comment:
s/virtual files/non-file URLs/ ? The terminology "virtual file" is highly overloaded and we support various kinds of virtual files.

Reply:
Updated, changed to blacklisted